### ISSUE TITLE
Cannot use tab (keyboard only) navigation to select tab filters

### ISSUE THREAD

**Issue by JamesDuncanFlindersUni** [sajari/website-search-integration#13]
Hi,

When navigating the website integration search interface, a user who is restricted to keyboard-only navigation is unable to select or interact with filter tabs.

For users that fit this profile, it is generally required that clickable elements, especially those that are not inputs or 'a' tags with 'href' values, have a tabindex value, or can be navigated to without a mouse.

`tabindex="0"` on each `.sj-tabs__tab` would be sufficient. 

Please see gif below demonstrating issue:

![sajari_keyboard_navigation_with_tabs](https://user-images.githubusercontent.com/47310558/53707109-6e11b880-3e7d-11e9-8175-cf6dfa88e21b.gif)
